runtime/src/main/jni/CallbackHandlers.cpp

```diff
@@ -317,8 +317,7 @@ void CallbackHandlers::CallJavaMethod(const Local<Object>& caller, const string&
         callerJavaObject = objectManager->GetJavaObjectByJsObject(caller);
         if (callerJavaObject.IsNull()) {
             stringstream ss;
-            ss << "No java object found on which to call \"" << methodName <<
-               "\" method. It is possible your Javascript object is not linked with the corresponding Java class. Try passing context(this) to the constructor function.";
+            ss << "Failed calling " << methodName << " on a " << className << " instance. The JavaScript instance no longer has available Java instance counterpart.";
```

Comment:
Because in JNI, the constructor call is just a method call, the most often mistake is to not pass the context when creating a native class (many of the constructor calls require context), so we decided to leave this message, because this mistake was made quite often, when you don't have d.ts for the native API.